packages/notifications-library/src/notifications_library/templates/on_share_project.email.content.txt

```diff
@@ -1,8 +1,8 @@
 Dear {{ user.first_name or user.user_name }},
 
-Great news! {{ sharer.user_name }} has shared a {{ product.ui.project_alias }} with you on {{ product.display_name }}.
+Great news! {{ sharer.user_name }} has shared a project with you on {{ product.display_name }}.
```

Comment:
is there no project.name? cause just "a project" is quite vague. just a thought.